src/main/java/levy/gameoflife/GameOfLife.java

```diff
@@ -66,19 +77,74 @@ public boolean isInBounds(int x, int y) {
         return x >= 0 && x < height && y >= 0 && y < width;
     }
 
-    public int getWidth() {
-        return width;
-    }
+    public void loadRle(String rleContent) {
```

Comment:
I think you had an RLE parser, why did you put this here?

Reply:
I wa playing around a lot in class because thought you wanted me to get rid of regex but once i realized was okay i ended up with this